### DISCUSSION TITLE
Allow Users to report issues

### DISCUSSION THREAD

**Discussion by 5drH64gDnDJb** [sct/overseerr#1263] [Feature Requests]
Could we get an option where a plex users can search for available media and report issues for that content. When I was using ombi, we could create custom categories that users can select and add comment, example:

-Wrong Episode
-Missing Episodes
-No Subtitles
-Audio Issues
-Video Issues
-Wrong Movie

Once the issue is reported, I would get an email saying new issue reported, i would mark the issue in progress and once the movie/tv show has been fixed, I can make a comment and close the issue. Once the issue is closed it would send an email to that plex users with my response / fix.

**Comment by TheCatLady** [comment]
Hi @5drH64gDnDJb! 👋🏻  This is planned. You can thumbs-up and subscribe to this issue for updates: https://github.com/sct/overseerr/issues/635

**Reply by 5drH64gDnDJb** [reply]
my bad, didnt see that, thank you very much!
